src/schema/v2/Alerts/index.ts

```diff
@@ -534,6 +537,9 @@ export const PartnerAlertsEdgeFields = {
         )
       }
 
+      // Make API call to fetch the first X collector profile records
+      const slicedUserIds = parent.user_ids.slice(0, 20)
```

Comment:
A more graphql/connection-y way to do this would be to rely on `first`, vs size (which is a foreign concept in connection-based cursoring). 

So something like: `first: args.first ?? 10` would do it

Reply:
Good idea ^^